authorization/invitation/model/invitation_model_service.go

```diff
@@ -2,41 +2,51 @@ package model
 
 import (
 	"context"
-
 	"fmt"
+
 	"github.com/fabric8-services/fabric8-auth/account"
 	"github.com/fabric8-services/fabric8-auth/authorization"
 	"github.com/fabric8-services/fabric8-auth/authorization/invitation"
-	invRepo "github.com/fabric8-services/fabric8-auth/authorization/invitation/repository"
-	permissionService "github.com/fabric8-services/fabric8-auth/authorization/permission/model"
-	"github.com/fabric8-services/fabric8-auth/authorization/repository"
-	resourceRepo "github.com/fabric8-services/fabric8-auth/authorization/resource/repository"
+	invitationrepo "github.com/fabric8-services/fabric8-auth/authorization/invitation/repository"
+	permissionsrv "github.com/fabric8-services/fabric8-auth/authorization/permission/model"
+	resource "github.com/fabric8-services/fabric8-auth/authorization/resource/repository"
+	resourcerepo "github.com/fabric8-services/fabric8-auth/authorization/resource/repository"
```

Comment:
This is being imported twice

Reply:
Fixed - https://github.com/fabric8-services/fabric8-auth/pull/453/commits/e1719ad7f74cc2056a1f0f689203fe9e68cd50f2